diff --git a/lib/chips/BQ25731RSN.circuit.tsx b/lib/chips/BQ25731RSN.circuit.tsx
index d72aa478513b4f07912cfe1317215c57e75df18b..20bd1e44df472a3b0656138349e34d1d5862f9f0 100644
--- a/lib/chips/BQ25731RSN.circuit.tsx
+++ b/lib/chips/BQ25731RSN.circuit.tsx
@@ -34,7 +34,7 @@ const pinLabels = {
   pin30: "BTST1",
   pin31: "HIDRV1",
   pin32: "SW1",
-  pin33: "Thermal_Pad",
+  pin33: "thermalpad",
 } as const;
 
 export const BQ25731RSN = (props: ChipProps<typeof pinLabels>) => (
